### ISSUE TITLE
Missing openrover-demo.repos, building on eloquent

### ISSUE THREAD

**Issue by JStech** [RoverRobotics/openrover-ros2#7]
The readme has a broken link to `openrover-demo.repos`. I was unable to find the file in the package history.

In the meantime, here are the repos and branches I had to clone to build on eloquent:
```
github.com/BehaviorTree/BehaviorTree.CPP ros2-devel
github.com/RoverRobotics/bno055_driver master
github.com/RoverRobotics/keystroke-ros2 master
github.com/ros-planning/navigation2 eloquent-devel
github.com/allenh1/rplidar_ros ros2
github.com/RoverRobotics-forks/serial-ros2 master
```

Also, in `openrover_demo/package.xml` I had to remove the `nav2_world_model` dependency, as it is no longer in `navigation2`.

**Comment by NathanDodson** [comment]
Yes I have the same issue. Any plan from RoverRobotics to fix this? The readme also sources dashing instead of eloquent.

**Comment by haizadtarik** [comment]
I think the correct link is [here](https://gist.github.com/rotu/0f29b7df4eb6134d4df3f0dce6b38f7e)

**Comment by NathanDodson** [comment]
It's nearly the right one, but it has dashing branches in it.

**Comment by rotu** [comment]
@haizadtarik that's the correct one from the time this was authored. I'll try to get this updated and in the repo ASAP.
